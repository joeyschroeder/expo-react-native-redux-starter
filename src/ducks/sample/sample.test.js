import { DEFAULT_STATE, resetSampleData, sampleReducer, updateSampleData } from './sample';

describe('sampleReducer', () => {
  describe('updateSampleData', () => {
    it('should add one to data', () => {
      const result = sampleReducer(DEFAULT_STATE, updateSampleData());
      expect(result).toMatchSnapshot();
    });
  });

  describe('resetSampleData', () => {
    it('should reset data', () => {
      const result = sampleReducer(DEFAULT_STATE, resetSampleData());
      expect(result).toMatchSnapshot();
    });
  });
});
