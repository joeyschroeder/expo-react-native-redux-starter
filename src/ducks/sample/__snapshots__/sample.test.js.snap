// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`sampleReducer resetSampleData should reset data 1`] = `
Object {
  "data": 0,
}
`;

exports[`sampleReducer updateSampleData should add one to data 1`] = `
Object {
  "data": 1,
}
`;
